partner_industry_parent/models/res_partner.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, _, api, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def _search(
+        self,
+        args,
+        offset=0,
+        limit=None,
+        order=None,
+        count=False,
+        access_rights_uid=None,
+    ):
+        if self._context.get("search_default_industry_id"):
+            args.append(
+                (
+                    "industry_id",
+                    "child_of",
+                    self._context["search_default_industry_id"],
+                )
+            )
+        return super()._search(
+            args,
+            offset=offset,
+            limit=limit,
+            order=order,
+            count=count,
+            access_rights_uid=access_rights_uid,
+        )
+
+    @api.model
+    def view_header_get(self, view_id, view_type):
+        if self._context.get("industry_id"):
+            return _(
```

Comment:
Is this right? I don't really know translations but I thought the rule was `_(<format_string>), args` not `_(<format_string>, args)`

Reply:
@gdgellatly Yes, Odoo introduced args to translation function in v16 I think. So, this is the correct way of doing it.